tendrl/ceph_integration/manager/manager.py

```diff
@@ -151,13 +150,13 @@ def main():
         config.get('ceph-integration', 'log_cfg_path'),
         config.get('ceph-integration', 'log_level')
     )
-    if sys.argv:
-        if len(sys.argv) > 1:
-            if "cluster-id" in sys.argv[1]:
-                cluster_id = sys.argv[2]
-                utils.set_tendrl_context(cluster_id)
 
-    m = CephIntegrationManager(utils.get_tendrl_context())
+    cluster_id = utils.get_tendrl_context()
+    if not cluster_id:
+        LOG.error("Could not find cluster_id")
```

Comment:
could not find cluster_id where? please add path where you are looking for cluster_id or add some logging to get_tendrl_context()